src/main/js/dynamic/loaded/users/UserManagement.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import ConfigurationLabel from 'component/common/ConfigurationLabel';
+import RoleTable from 'dynamic/loaded/users/RoleTable';
+import UserTable from 'dynamic/loaded/users/UserTable';
+import * as DescriptorUtilities from 'util/descriptorUtilities';
+import CollapsiblePane from 'component/common/CollapsiblePane';
+
+class UserManagement extends Component {
+
+    constructor(props) {
+        super(props);
+
+        const descriptor = DescriptorUtilities.findDescriptorByNameAndContext(this.props.descriptors, DescriptorUtilities.DESCRIPTOR_NAME.COMPONENT_USERS, DescriptorUtilities.CONTEXT_TYPE.GLOBAL)[0];
+
+        this.state = {
+            descriptor: descriptor
+        };
+    }
+
+    checkPermissions(operation) {
```

Comment:
We do this in multiple places we should probably add a utility to check permissions and put it in the util directory like the FieldModel utilities.